packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableEditor.ts

```diff
@@ -219,4 +232,24 @@ export default class TableEditor {
         this.disposeTableResizer();
         this.onFinishEditing();
     };
+
+    private onSelect = (table: HTMLTableElement) => {
+        this.editor.focus();
+        if (table) {
+            const rows = table.rows.length - 1;
+            const lastCellIndex = table.rows.item(rows).cells.length - 1;
```

Comment:
what if last row has merged cell?

suggest use VTable to get column count

Reply:
Sounds good adding VTable here.